test/pytest/hidra/test_transfer.py

```diff
@@ -0,0 +1,392 @@
+import logging
```

Comment:
Tests are a bit too deep into HiDRA for me to have any meaningful opinion on them ;)